src/components/NavBar.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from './NavBar.module.css';
```

Comment:
This component (and css file) are in the wrong directory. They should be in `src/core/components/`